Dockerfile.builder

```diff
@@ -27,7 +27,7 @@ RUN chmod 777 -R ${HOME}
 
 RUN apt-get update \
     && apt-get install --no-install-recommends -y -q \
-    curl wget git unzip bzip2 jq
+    curl wget git unzip bzip2 jq libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```

Comment:
I'd put these on a separate line with a comment that these are for cypress (and a link to the doc)

Reply:
Fixed